test/Microsoft.Azure.WebJobs.Host.EndToEndTests/Scale/ScaleHostEndToEndTests.cs

```diff
@@ -11,17 +17,11 @@
 using Microsoft.Extensions.Logging;
 using Microsoft.Extensions.Options;
 using Newtonsoft.Json.Linq;
-using System;
-using System.Collections.Generic;
-using System.IO;
-using System.Linq;
-using System.Text;
-using System.Threading;
-using System.Threading.Tasks;
 using Xunit;
 
 namespace Microsoft.Azure.WebJobs.Host.EndToEndTests.Scale
 {
+    [Trait(TestTraits.CategoryTraitName, TestTraits.ScaleMonitoring)]
```

Comment:
Using traits like this make it easy to run just the tests you care about by feature in VS :)